flocker/node/agents/functional/test_ebs.py

```diff
@@ -163,6 +164,31 @@ def test_next_device_in_use(self):
                                        {u"/dev/sdf"})
         self.assertEqual(result, u"/dev/sdg")
 
+    @capture_logging(
+        assertHasMessage, IN_USE_DEVICES, {
+            'devices': [u'/dev/sda1', u'/dev/sdf']
+        },
+    )
+    def test_in_use_devices_log(self, logger):
+        """
+        Test that attached device shows up as being in use during subsequent
+        ``attach_volume``.
+        """
```

Comment:
Nit: You never need to say "Test that" in a test docstring. You can just described the behaviour, e.g. "The attached device shows up ..."

https://jml.io/test-docstrings/